### PR TITLE
3.4 release blog

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

https://github.com/prettier/prettier/issues/16850

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: changelog_unreleased/blog-post-intro.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 3.4: Fixes a lot of bugs"
```
Comment (kachkaev) [inline changelog_unreleased/blog-post-intro.md:4]:
```suggestion
title: "Prettier 3.4: A lot of bug fixes"
```

Optional: I could not find a blog post title starting with a verb, but there was https://prettier.io/blog/2019/06/06/1.18.0 for example

File: website/blog/2024-11-24-3.4.0.md [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+### JavaScript
+
+#### Fix template literal print with array ([#13315](https://github.com/prettier/prettier/pull/13315) by [@fisker](https://github.com/fisker), [@syi0808](https://github.com/syi0808))
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+const string = `${[[1, 2], [3, 4]]}`
+
+// Prettier 3.2
```
Comment (kachkaev) [inline website/blog/2024-11-24-3.4.0.md:24]: I believe this should be
```suggestion
// Prettier 3.3
```
here and below

File: website/blog/2024-11-24-3.4.0.md [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+  $newKey: (
+    $key:
+      (
+        $theme-name: $value,
+      ),
+  );
+}
+```
+
+#### Fixed wrong trailing comma position after comment in SCSS ([#16617](https://github.com/prettier/prettier/pull/16617) by [@Ma-hawaj](https://github.com/Ma-hawaj), [@fisker](https://github.com/fisker))
```
Comment (kachkaev) [inline website/blog/2024-11-24-3.4.0.md:415]:
```suggestion
#### Fix wrong trailing comma position after comment in SCSS ([#16617](https://github.com/prettier/prettier/pull/16617) by [@Ma-hawaj](https://github.com/Ma-hawaj), [@fisker](https://github.com/fisker))
```

File: website/blog/2024-11-24-3.4.0.md [hunk 4]
```diff
@@ -0,0 +1,796 @@
...
+<!-- Prettier 3.4 -->
+<template>
+  <button @click="点击事件">点击!</button>
+  <button @click="onClick">Click!</button>
+</template>
+```
+
+### Markdown
+
+#### Removed excessive spaces after line prefixes for unordered lists in Markdown ([#15526](https://github.com/prettier/prettier/pull/15526) by [@TomasLudvik](https://github.com/TomasLudvik))
```
Comment (kachkaev) [inline website/blog/2024-11-24-3.4.0.md:465]:
```suggestion
#### Remove excessive spaces after line prefixes for unordered lists in Markdown ([#15526](https://github.com/prettier/prettier/pull/15526) by [@TomasLudvik](https://github.com/TomasLudvik))
```

**Review by kachkaev — COMMENTED** [review]
Nice! Left a couple of small suggestions.

Blog post preview:
https://deploy-preview-16851--prettier.netlify.app/blog/2024/11/25/3.4.0

**Review by fisker — APPROVED** [review]

**Review by kachkaev — APPROVED** [review]

File: website/blog/2024-11-25-3.4.0.md
Comment (kachkaev) [inline website/blog/2024-11-25-3.4.0.md:1]: N.B. We’ll need to change the date if we don’t manage to release 3.4 by the end of today [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth)

**Pull request merged**